### ISSUE TITLE
datasette-sqlite-vec plugin fails to register hook

### ISSUE THREAD

**Issue by simonw** [asg017/sqlite-vec#60]
The `datasette-sqlite-vec` plugin isn't currently registering the plugin hooks correctly. Should be a small fix.

**Comment by simonw** [comment]
I think the bug is over in `sqlite-dist` - somewhere around here: https://github.com/asg017/sqlite-dist/blob/f4e43b3a708e23fde716fc2cb04123fc1ff1f522/src/pip.rs#L316

**Comment by simonw** [comment]
This workaround works:
```bash
datasette --load-extension "$(python -c 'print(__import__("sqlite_vec").loadable_path())')"
```

**Comment by asg017** [comment]
@simonw Datasette and sqlite-utils bindings have been fixed in `v0.1.1`, let me know if it doesnt work for you! https://github.com/asg017/sqlite-vec/releases/tag/v0.1.1
